cdc/sinkv2/eventsink/txn/worker.go

```diff
@@ -33,12 +35,13 @@ type worker struct {
 	stopped chan struct{}
 	wg      sync.WaitGroup
 	backend backend
+	errCh   chan<- error
```

Comment:
Please add todo here, remove the error chan

Reply:
OK. I will update it in the next PR.